bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HBaseMutationCoder.java

```diff
@@ -41,31 +43,35 @@
 public class HBaseMutationCoder extends AtomicCoder<Mutation> implements Serializable {
 
   private static final long serialVersionUID = -3853654063196018580L;
+  private static final PutAdapter PUT_ADAPTER = new PutAdapter(Integer.MAX_VALUE);
```

Comment:
PutAdapter's parameter is Configuration driven.  The parameter is max key size, which could be defined in hbase-site.xml